src/OrbitGl/OrbitApp.cpp

```diff
@@ -1465,8 +1465,7 @@ bool OrbitApp::IsCaptureConnected(const CaptureData& capture) const {
   const ProcessData* capture_process = capture.process();
   ORBIT_CHECK(capture_process != nullptr);
 
-  return selected_process->pid() == capture_process->pid() &&
-         selected_process->full_path() == capture_process->full_path();
+  return selected_process->pid() == capture_process->pid();
```

Comment:
The function is now (that we actually open a loaded capture in a new window, see comment above) a bit confusion. It would be probably even enough to check if we do have a selected process. In this way the weaker check now should be completely fine. But I think the comment should be updated.

Reply:
Thanks for the reviews! The comment is pretty confusing I agree, I removed it altogether. The code is self explanatory to me now. I don't really want to spend time on this tbh, simply fix an annoying bug :-)